modules/openy_gc_auth/src/GCUserAuthorizer.php

```diff
@@ -72,11 +73,22 @@ public function authorizeUser($name, $email, array $extra_data = []) {
         $account->save();
       }
     }
+    // List of roles to redirect user login page.
+    $userRolesArray = [
+      'administrator',
+      'virtual_ymca_editor',
+    ];
+    // Redirecting user login page.
+    foreach ($userRolesArray as $role) {
+      if ($account->hasRole($role)) {
+        $response = new RedirectResponse('/user/login', 301);
```

Comment:
Please, add this code above:
`$loginUrl = Url::fromRoute('user.login');`
and replace this line:

```suggestion
        $response = new RedirectResponse($loginUrl, 301);
```
It will be a more stable approach. I had many projects in the past when we were renaming user login/register routes to avoid spam bots attacks. In your case, it is hard-coded.

Reply:
@anpolimus  I have updated the code as per you suggestion, Can you please review.